integration_tests/serverless-plugin.yml

```diff
@@ -0,0 +1,116 @@
+service: integration-plugin
+
+provider:
+  name: aws
+  environment:
+    DD_INTEGRATION_TEST: true
+    DD_API_KEY: ${env:DD_API_KEY}
+
+layers:
+  node10:
+    package:
+      artifact: ../.layers/datadog_lambda_node10.15.zip
+  node12:
+    package:
+      artifact: ../.layers/datadog_lambda_node12.13.zip
+
+functions:
+  # async-metrics
+  async-metrics_node10_with_plugin:
+    handler: send-metrics.handle
+    runtime: nodejs10.x
+    layers:
+      - { Ref: Node10LambdaLayer }
+    environment:
+      DD_FLUSH_TO_LOG: true
+      WITH_PLUGIN: true
+
+  async-metrics_node12_with_plugin:
+    handler: send-metrics.handle
+    runtime: nodejs12.x
+    layers:
+      - { Ref: Node12LambdaLayer }
+    environment:
+      DD_FLUSH_TO_LOG: true
+      WITH_PLUGIN: true
+
+  # sync-metrics
+  sync-metrics_node10_with_plugin:
+    handler: send-metrics.handle
+    runtime: nodejs10.x
+    layers:
+      - { Ref: Node10LambdaLayer }
+    environment:
+      WITH_PLUGIN: true
+
+  sync-metrics_node12_with_plugin:
+    handler: send-metrics.handle
+    runtime: nodejs12.x
+    layers:
+      - { Ref: Node12LambdaLayer }
+    environment:
+      WITH_PLUGIN: true
+
+  # http-requests
+  http-requests_node10_with_plugin:
+    handler: http-requests.handle
+    runtime: nodejs10.x
+    layers:
+      - { Ref: Node10LambdaLayer }
+    environment:
+      WITH_PLUGIN: true
+
+  http-requests_node12_with_plugin:
+    handler: http-requests.handle
+    runtime: nodejs12.x
+    layers:
+      - { Ref: Node12LambdaLayer }
+    environment:
+      WITH_PLUGIN: true
+
+  # process-input-traced
+  process-input-traced_node10_with_plugin:
+    handler: process-input-traced.handle
+    runtime: nodejs10.x
+    layers:
+      - { Ref: Node10LambdaLayer }
+    environment:
+      WITH_PLUGIN: true
+      DD_FLUSH_TO_LOG: true
+
+  process-input-traced_node12_with_plugin:
+    handler: process-input-traced.handle
+    runtime: nodejs12.x
+    layers:
+      - { Ref: Node12LambdaLayer }
+    environment:
+      WITH_PLUGIN: true
+      DD_FLUSH_TO_LOG: true
+
+  # http-errors
+  http-errors_node10_with_plugin:
+    handler: http-errors.handle
+    runtime: nodejs10.x
+    layers:
+      - { Ref: Node10LambdaLayer }
+    environment:
+      WITH_PLUGIN: true
+
+  http-errors_node12_with_plugin:
+    handler: http-errors.handle
+    runtime: nodejs12.x
+    layers:
+      - { Ref: Node12LambdaLayer }
+    environment:
+      WITH_PLUGIN: true
+
+plugins:
+  - serverless-plugin-datadog
+
+custom:
+  datadog:
+    flushMetricsToLogs: true
+    enableXrayTracing: true
+    addLayers: false
+    enableDDTracing: false
```

Comment:
When I enable this I'm getting non-deterministic results. Trace logs are showing partially for some lambdas. Created this ticket to track this and dig a bit more https://datadoghq.atlassian.net/browse/SLS-690